modules/gentemplates/rapidml-swagger/src/test/java/com/reprezen/genflow/rapidml/swagger/test/MultilineCommentTest.xtend

```diff
@@ -122,7 +121,7 @@ class MultilineCommentTest extends SwaggerGenTestBase {
 		assertEquals(expected.trim, processed)
 	}
 
-	@Test @Ignore
+	@Test
```

Comment:
@nbhusare , it looks like you have accidentally overwritten the `@ignore` annotations here in MultilineCommentTest. Please reinstate.